plugins/outputs/microsoft_fabric/event_house_test.go

```diff
@@ -0,0 +1,106 @@
+package microsoft_fabric
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/influxdata/telegraf/plugins/common/adx"
+	"github.com/influxdata/telegraf/testutil"
+)
+
+func TestEventHouseConnectSuccess(t *testing.T) {
+	tests := []struct {
+		name     string
+		endpoint string
+		database string
+	}{
+		{
+			name:     "valid configuration",
+			endpoint: "addr=https://example.com",
+			database: "testdb",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Setup plugin
+			plugin := &eventhouse{
+				connectionString: tt.endpoint,
+				Config: adx.Config{
+					Database: tt.database,
+				},
+				log: testutil.Logger{},
+			}
+			require.NoError(t, plugin.init())
+
+			// Check for successful connection and client creation
+			require.NoError(t, plugin.Connect())
+			require.NotNil(t, plugin.client)
```

Comment:
Missing cleanup - resources not closed after test
```suggestion
			// Check for successful connection and client creation
	                require.NoError(t, plugin.Connect())
	                require.NotNil(t, plugin.client)
	                
	                // Clean up resources
	                require.NoError(t, plugin.Close())
```